src/StatsigClient.ts

```diff
@@ -287,6 +287,14 @@ export default class StatsigClient implements IHasStatsigInternal, IStatsig {
     );
   }
 
+  public getEvaluationDetails(): EvaluationDetails {
+    const details = this.store.getGlobalEvaluationDetails();
+    if (details.reason === EvaluationReason.Unrecognized) {
+      details.reason = EvaluationReason.Uninitialized;
+    }
```

Comment:
why is this conversion needed?

Reply:
This should be removed, nice catch.  From a previous attempt